week1/solutin-to-homework.py

```diff
@@ -0,0 +1,87 @@
+# 1. Write a python script to print your name and age
+Name = input("Enter  Name:")
+age = input("Enter age:")
+print(f'Hello {Name.capitalize()} \n You are {age} years old')
+
+# 2. Create a list of your 5 favorite movies and store it in the variable
+favourite_movies = []
+for i in range(5):
+
+    movie = input(f'enter your {i + 1} favourite movie:')
+    favourite_movies.append(movie)
+
+print(f'my Favourite movie are:{favourite_movies}')
+
+# 3.Write a Python program to display the first and last colors from the following list . color_list = ["Red","Green","White" ,"Black"]
+color_list = ["Red", "Green", "White", "Black"]
+print(f'{color_list[0]} is first color {color_list[-1]} is last color from the above list.')
+
+# 4.Write a Python script to add a key to a dictionary
+# Sample Dictionary : {0: 10, 1: 20}
+# Expected Result : {0: 10, 1: 20, 2: 30}
+dict = {0: 10, 1: 20}
+dict.update({2: 30})
+print(f'updated dict:{dict}')
+
+# 5. Write a Python program to calculate body mass index.
+def calculate_BMI(height, weight):
+    return round(weight / (height * height), 2)
```

Comment:
`height * height` is right but there is a more elegant way of writing it. Look for the power to operator